src/vorta/application.py

```diff
@@ -325,7 +326,9 @@ def check_failed_response(self, result: Dict[str, Any]):
             if returncode == 1:
                 # warning
                 msg.setIcon(QMessageBox.Icon.Warning)
-                text = self.tr('Borg exited with a warning message. See logs for details.')
+                text = translate(
+                    'VortaApp', 'Borg exited with warning status (rc 1). See <a href="{0}">logs</a> for details.'
```

Comment:
considering that the translations need an update now anyway, guess we could fix the original string:

"See **the** logs for details."

Also:

Do we have multiple logs or why the plural?

And even if we have multiple - for that specific backup, only one of them is meant, right?

Reply:
> Do we have multiple logs or why the plural?

We have a single log spread over multiple files (RotatingFileHandler).